mufasa/master_fitter.py

```diff
@@ -230,7 +230,11 @@ def refit_swap_2comp(reg, snr_min=3, fit_kw=None):
 
 
 
-def refit_2comp_wide(reg, snr_min=3, method='residual', planemask=None, fit_kw=None):
+def refit_2comp_wide(reg, snr_min=3, planemask=None, **kwargs):
```

Comment:
I think we can keep the `method` argument separately from `**kwargs`. It doesn't look like the `method` keyword is found anywhere downstream. Neither `pyspeckit.cubes.SpectralCube` nor `pyspeckit.spectrum.fitters` seem to use `method` as an argument keyword. For that reason, I don't think we need a key called `wide_refit_method` either.